CHANGELOG.md

```diff
@@ -2,6 +2,26 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [19.0.0] - 2019-02-13
+### Added
+- added `running_tasks` property to `Context`
+- added `WorkerShutdownDuringTask` exception
```

Comment:
Hmm, I know that this exception is "visible" to implementers, but it should never be raised by them, nor will they every catch a raised `WorkerShutdownDuringTask`. 

I suppose splitting the scriptworker client and worker will resolve this :)

Reply:
tree, signing, shipit, pushsnap, bouncer, beetmover, addon all show up in a `ag scriptworker *script | grep import | grep -v client`